cts-java/src/test/java/org/ga4gh/cts/api/TestData.java

```diff
@@ -148,6 +148,21 @@ private TestData() {
     public static final String CALL_SET_NAME = "HG00096";
 
     /**
+     * The BioSample name is used to test searching by BioSample name functionality.
+     */
+    public static final String BIOSAMPLE_NAME = "HG00096";
+
+    /**
+     * The VariantSet name in the test data.
+     */
+    public static final String VARIANTSET_NAME = "1kgPhase3";
+
+    /**
+     * The individual name in the test data.
+     */
+    public static final String INDIVIDUAL_NAME = "HG00096";
```

Comment:
We may want, as part of a larger overhaul of the compliance test dataset, to throw in a couple of individuals here, one with no biosamples, and one with multiple associated biosamples.